content/src/main/content/jcr_root/apps/acs-commons/authoring/accordion/rte-accordion.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * #%L
+ * ACS AEM Commons Package
+ * %%
+ * Copyright (C) 2016 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+(function($, CUI){
+    var RTEAccordion = {
+        GROUP: "accordion",
+        FEATURE: "rte-accordion"
+    };
+ 
+    // Extend toolbar to register accordion icon
+    CUI.rte.ui.cui.CuiToolbarBuilder = new Class({
```

Comment:
I tried removing this class and adding `tbGenerator.registerIcon(RTEAccordion.GROUP + "#" + RTEAccordion.FEATURE, "coral-Icon coral-Icon--feed");` to `initializeUI` in `RTEAccordion.TouchUIAccordionPlugin` but can't seem to get the icon to show up in the fullscreen RTE anymore.

Are there any resources/documentation online that you could point me to for this?